src/config.ts

```diff
@@ -0,0 +1,105 @@
+import {ShopifyError, UninitializedConfigError, PrivateAppError} from './error';
```

Comment:
This is probably the most important place to check (and its test file). Anything else is just search / replacing for the name change.